staging/src/k8s.io/dynamic-resource-allocation/resourceslice/resourceslicecontroller.go

```diff
@@ -205,10 +311,10 @@ func (c *Controller) initInformer(ctx context.Context) error {
 
 	// We always filter by driver name, by node name only for node-local resources.
 	selector := fields.Set{
-		resourceapi.ResourceSliceSelectorDriver:   c.driver,
+		resourceapi.ResourceSliceSelectorDriver:   c.driverName,
 		resourceapi.ResourceSliceSelectorNodeName: "",
 	}
-	if c.owner.APIVersion == "v1" && c.owner.Kind == "Node" {
+	if c.owner != nil && c.owner.APIVersion == "v1" && c.owner.Kind == "Node" {
```

Comment:
Does this mean that I can now omit passing an `owner` if it doesn't make much sense (e.g. for non-node-local resources)?

Reply:
Yes.